nuitka/__past__.py

```diff
@@ -94,6 +94,10 @@ def total_ordering(cls):
 
         return cls
 
+try:
```

Comment:
Actually I think this ought to use `if str is bytes:` instead of relying on the import error, much like this one:

```
if str is bytes:
    from urllib import (  # pylint: disable=I0021,import-error,no-name-in-module
        urlretrieve,  # @UnresolvedImport @UnusedImport
    )
else:
    from urllib.request import (  # pylint: disable=I0021,import-error,no-name-in-module
        urlretrieve,  #  @Reimport @UnresolvedImport
    )
```

Also not the comments, PyDev and PyLint will otherwise complain.

The reason is that sometimes people make backports of Python3 package names or forward ports of Python2 package names, and it's nicer if you express which version has which one. Since it's 3.3 or higher, for Nuitka that is Python3, and `str is bytes` is a very good test for that.

Reply:
Indeed, done :+1: